### PR TITLE
Rename to App Store

### PR DESCRIPTION
**Pull request by rfun**
This should hopefully complete the rename for the warehouse to app store. there might be too many files to review so the easiest way to test would be to pull this branch `rename` and try installing it locally. 

Once this is merged I will push out a build with the new name after resolving the merge conflicts.

### CODE REVIEW

**Comment by rileyhales** [comment]
Looks good. I fresh installed and built the docs. Everything looks ok. 1 thing to be fixed.
- On the app's page in a portal, the question mark button to open the help model doesn't do anything. Should fix this while we're here.

3 other things that maybe don't have to be fixed in this pull request:
- The screenshots in the docs show the old name. We should fix that before this project is "finished" and other people need to start consulting the docs for help.
- Also for the docs, there are several blank pages. Some of these pages could be moved from separate pages to subheadings on existing pages.
- The example code on docs/github install api/github action example has a comment about changing the URL in the yml which should be changed. This is higher priority than the other 2 but i would suggest it should happen in a separate pull request to synchronize the docs changes with the updated installation on tethys-staging.

**Comment by rileyhales** [comment]
Also- the link to readthedocs still doesn't work. We currently have app-store.readthedocs and tethys-app-store.readthedocs listed on the readme but neither work. This needs to be added to readthedocs.

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 
-[![Build Status](https://github.com/BYU-Hydroinformatics/warehouse/workflows/main/badge.svg)](https://github.com/BYU-Hydroinformatics/warehouse/actions)
+[![Build Status](https://github.com/BYU-Hydroinformatics/app_store/workflows/main/badge.svg)](https://github.com/BYU-Hydroinformatics/app_store/actions)
 
 A Tethys application to discover, install and manage Tethys applications for a Tethys Portal. This project is developed by Rohit Khattar as a part of a Doctorate degree in Civil and Environmental Engineering at Brigham Young University.
 
 ## Documentation:
 
-See: [ReadTheDocs](https://tethys-app-warehouse.readthedocs.io/en/latest) : https://tethys-app-warehouse.readthedocs.io
+See: [ReadTheDocs](https://app-store.readthedocs.io/en/latest/) : https://app-store.readthedocs.io
```
Comment (rileyhales) [inline README.md:11]: this url to readthedocs does not work

File: docs/github-install.rst [hunk 2]
```diff
@@ -96,6 +96,7 @@ Github action example
           uses: fjogeleit/http-request-action@master
           with:
             method: 'POST'
+            # The URL below might need to be updated since the warehouse is being renamed to app store
```
Comment (rileyhales) [inline docs/github-install.rst:99]: This needs to be changed but coordinate the rename here with the install of the renamed app on tethys-staging

File: install.yml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 version: 0.1
```
Comment (rileyhales) [inline install.yml:1]: This would be a good time to start versioning. You could call this 1.0?

File: setup.py [hunk 4]
```diff
@@ -29,13 +29,13 @@ def find_resource_files(directory, relative_to=None):
 
 setup(
     name=release_package,
-    version='0.3.1',
-    description='Tethys Application Warehouse',
+    version='1.0.0',
```
Comment (rileyhales) [inline setup.py:32]: Oh, i see you're tracking version numbers here. syncing with the install.yml would be a good idea but i don't care that much.

**Review by rileyhales — APPROVED** [review]

**Comment by rileyhales** [comment]
I approved but i would really recommend we put this on readthedocs asap. i believe you will be required to use the url tethysapp-tethys_app_store.readthedocs.io because thats the name of the repository at the time you submit it. So there will be more changes to the pr to get the urls to the docs correct. @rfun you could ask nathan if he would let you subdomain tethysplatform.org e.g. you can put your docs at app-store.tethysplatform.org or something like that. You just need whoever owns the domain to add a cname or A record

**Comment by rfun** [comment]
> I approved but i would really recommend we put this on readthedocs asap. i believe you will be required to use the url tethysapp-tethys_app_store.readthedocs.io because thats the name of the repository at the time you submit it. So there will be more changes to the pr to get the urls to the docs correct. @rfun you could ask nathan if he would let you subdomain tethysplatform.org e.g. you can put your docs at app-store.tethysplatform.org or something like that. You just need whoever owns the domain to add a cname or A record

@rileyhales : Thanks for all the comments. I agree with most of these suggestions and will work on getting them resolved later today.

**Pull request merged**